java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -880,6 +872,11 @@ public void functionLoad_and_functionList(boolean singleNodeRoute) {
             }
         }
 
+        // load new lib and delete it - first lib remains loaded
+        String anotherLib = generateLuaLibCode("anotherLib", List.of("anotherFunc"));
+        assertEquals("anotherLib", clusterClient.functionLoad(anotherLib, true, route).get());
+        assertEquals(OK, clusterClient.functionDelete("anotherLib", route).get());
+
```

Comment:
Do we want to add a test asserting that an exception is thrown if you try and delete a function that doesn't exist or it is not necessary?

Reply:
Added in 462feb54